test/max.test.js

```diff
@@ -0,0 +1,8 @@
+import test from 'ava'
+import {max} from '../src'
```

Comment:
Oh! Got it! I done XD
Does import file by this way could have better performance? :)